examples/experimental/i3s-17-and-debug/bounding-volume-layer.js

```diff
@@ -91,19 +89,33 @@ export default class BoundingVolumeLayer extends CompositeLayer {
     return geometry;
   }
 
-  _generateMesh(tile) {
-    if (tile.header.obb || tile.boundingVolume instanceof OrientedBoundingBox) {
-      return this._generateCubeMesh(tile);
+  _generateMesh(tile, boundingVolumeType) {
+    switch (boundingVolumeType) {
+      case 'CubeGeometry':
```

Comment:
Perhaps always generate both and just choose the one we need when rendering?

Reply:
Ok, I'll create another PR to fix it